components/equihash/CHANGELOG.md

```diff
@@ -6,6 +6,8 @@ and this library adheres to Rust's notion of
 [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Changed
+- MSRV is now 1.66.0.
```

Comment:
Most of the `components/` crates use their own MSRVs as they are much more stable and rarely updated (note they do not use `rust-version.workspace = true`, so the top-level workspace MSRV change did not alter them). Revert these changes.
```suggestion
```